Apps/phsplunkoncall/splunkoncall.json

```diff
@@ -1,11 +1,11 @@
 {
     "appid": "623de41b-eac0-4a4a-970c-974e8d7ac2cb",
```

Comment:
The App ID needs to change too.

Reply:
@brian-phantom We have not changed the vendor and product name of the app. Do I still need to change the app id?

Reply:
I had a discussion with @brian-phantom, and decided that we don't need to change the appid.